src/vsr/client.zig

```diff
@@ -274,7 +274,7 @@ pub fn Client(comptime StateMachine_: type, comptime MessageBus: type) type {
         }
 
         fn on_pong(self: *Self, pong: *const Message) void {
-            assert(pong.header.command == .pong);
+            assert(pong.header.command == .pong_client);
```

Comment:
`fn on_pong_client` then?

Reply:
relatedly, while understanding view change protocol, the `start_view` / `start_view_change` naming tripped me so many times. It's too easy to search for `start_view_change_something`  and end up looking at the `start_view` code instead. 

If we follow through with our MSB naming, it probably should be:

```
viewchange_start
viewchange_do
view_start
request_view_start.
```

Reply:
So, yeah, given that this is client, I think `on_pong` is ok. Though, I'd still prefer `on_pong_client` perhaps for consistency.